ApiBundle/Controller/NodeController.php

```diff
@@ -168,9 +168,14 @@ public function listNodeByAuthorAndSiteIdAction($published)
     public function listTreeNode($siteId)
     {
         $nodes = $this->get('open_orchestra_model.repository.node')->findLastVersionByType($siteId);
-        $orderedNodes = $this->get('open_orchestra_display.manager.tree')->generateTree($nodes);
 
-        return $this->get('open_orchestra_api.transformer_manager')->get('node_tree')->transform(end($orderedNodes));
+        if (!empty($nodes)) {
```

Comment:
you could invert the test by writing if (empty($nodes)) { then return array(); }